### DISCUSSION TITLE
I made DBC 3.4.1 work on Windows 7

### DISCUSSION THREAD

**Discussion by Forbirdden** [aiko-chan-ai/DiscordBotClient#180] [Show and tell]
I made new version of DBC work on Windows 7/8/8.1 without any troubles. it takes 17 hours
[Check my hard work](https://github.com/Forbirdden/DiscordBotClient-OldWindows)

**Comment by aiko-chan-ai** [comment]
![460716660_562821652944409_6229513925471007210_n](https://github.com/user-attachments/assets/0daee175-f5cc-47cc-8228-ce9b95e13d53)

**Comment by aiko-chan-ai** [comment]
It seems like it is using version 302xxx of Discord.

**Reply by Forbirdden** [reply]
Yes, it is, DBC 3.4.1, uses same
![image](https://github.com/user-attachments/assets/9b844c14-a17a-43e2-85c4-53c73826476a)
![image](https://github.com/user-attachments/assets/475c8629-f116-4b98-95ff-cab2eb389d5c)
